mk/board_native.mk

```diff
@@ -38,6 +38,13 @@ DEFINES        += -DPERF_LTTNG=1
 LIBS           += -llttng-ust -ldl
 endif
 
+HAVE_SYSTEMD=0
+
+ifeq ($(HAVE_SYSTEMD),1)
```

Comment:
I think this should be HAVE_LIBSYSTEMD. And you should update the waf build system to check for its presence by using conf.check()